avalon/manage.py

```diff
@@ -1,7 +1,9 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python2
```

Comment:
Unintended change.

(BTW, I'm pretty sure the live version is actually running on Python 2 since I haven't touched it in a while. I think this should work fine on either, but, of course, I should transition to running on Python 3.)